demo/px-map-control-zoom-demo.html

```diff
@@ -1,6 +1,7 @@
 <!-- Common imports -->
 <link rel="import" href="../../polymer/polymer.html" />
-<link rel="import" href="../../px-polymer-font-awesome/px-polymer-font-awesome.html" />
+<link rel="import" href="../px-icon-set/px-icon-set.html"/>
```

Comment:
I don't think you need to import px-icon for all those demos since it is loaded by px-map